apriltags_ros/src/apriltag_detector.cpp

```diff
@@ -68,7 +76,16 @@ AprilTagDetector::~AprilTagDetector(){
   image_sub_.shutdown();
 }
 
+void AprilTagDetector::enableCb(const std_msgs::Bool& msg) {
+  enabled_ = msg.data;
```

Comment:
It may not serialize it over the wire, but it's still doing some work to put it into the ros image publisher format.